hypothesis-python/docs/changes.rst

```diff
@@ -4,22 +4,11 @@ Changelog
 
 This is a record of all past Hypothesis releases and what went into them,
 in reverse chronological order. All previous releases should still be available
-on pip.
+on `PyPI <https://pypi.org/project/hypothesis/>`__.
 
-Hypothesis APIs come in three flavours:
 
-* Public: Hypothesis releases since 1.0 are `semantically versioned <https://semver.org/>`_
-  with respect to these parts of the API. These will not break except between
-  major version bumps. All APIs mentioned in this documentation are public unless
-  explicitly noted otherwise.
-* Semi-public: These are APIs that are considered ready to use but are not wholly
-  nailed down yet. They will not break in patch releases and will *usually* not break
-  in minor releases, but when necessary  minor releases may break semi-public APIs.
-* Internal: These may break at any time and you really should not use them at
-  all.
-
-You should generally assume that an API is internal unless you have specific
-information to the contrary.
+Hypothesis 4.x
+==============
```

Comment:
Do these sections have any impact on release automation?

(Specifically, the part that automatically updates `changes.rst` with details of the new release.)

Reply:
Nope, it searches for the link anchor below and insert above that, below the new heading.